### ISSUE TITLE
Update index.html to connect listen for server events

### ISSUE THREAD

**Issue by c3ho** [Seneca-CDOT/telescope#461]
Let's try using Server Sent Events.

In another PR, we also need to update our frontend index.html to connect to this route, and listen for these updates.

**Comment by c3ho** [comment]
Complete message here from #314 

We need a way to signal to the browser that new posts are available to be downloaded. When our feedQueue completes processing all the posts, we need to listen for an event, and then trigger a message to the browser.

Let's try using Server Sent Events.

In another PR, we also need to update our frontend index.html to connect to this route, and listen for these updates.

**Comment by c3ho** [comment]
@humphd when you mean index.html... you're talking about main.js file right?

**Comment by humphd** [comment]
@c3ho correct.

**Comment by c3ho** [comment]
Alright, I got an issue. I played around a bit, but I'm not sure of the exact implementation.
I'm unable to include server-sent-events.js into main.js as a require (forgive me as it has been a while since I've done any front end)

I rewrote the function here to play around with it
```
// Entry point: document loaded
$(() => {
  getPostsPage();
  const source = new EventSource('/feed-updates');
  source.onmessage = function(e) {
    console.log('received event', e);
    // getPostsPage()
  };
});
```
Should I just call getPostsPage() again right after the console.log line?

**Comment by c3ho** [comment]
Update, I tried the getPostsPage() within the source.onmessage, it worked. It's a rough ride though as it'll keep update the page non stop while it processes all the feeds. I tried to put a timeout of 30s
like so to delay the getPostsPage() by 30s
```
  setTimeout(
    (source.onmessage = function(e) {
      console.log('received event', e);
      getPostsPage();
    }),
    30000
  );
```
Didn't work, still updating on every process. I'd imagine this wouldn't be as annoying when all the current posts have been processed.
